File: Lecture004 Solving Pattern Questions (Part-2)/pattern7.py

class solution():
    def pattern(n):
        row=1
        while(row<=n):
            col=1
            while(col<=n):
                str=ord('A')+row-1
                print(chr(str),end=" ")
                col+=1
            print()
            row+=1
        # ch="A"
        # print(ord(ch))

            

if __name__ == '__main__':
    ob=solution
    num = int(input("Enter Number = "))
    ob.pattern(num)

File: 004_Solving_Pattern_Questions_(Part-2)/pattern7.cpp
#include<iostream>
using namespace std;

int main() {

    int n;
    cin>>n;

    int row = 1;

    while(row <= n) {

        int col = 1;

        while(col <= row) {
            char ch = ('A' + row + col - 2);
            cout<<ch;
            col = col + 1;
        }
        cout<<endl;
        row = row + 1;

    }
    return 0;
}